DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -198,6 +207,12 @@ class CaloTower : public reco::LeafCandidate {
    GlobalPoint emPosition_;
    GlobalPoint hadPosition_;
 
+  //hcal limits in CT continuous ieta (from CaloTowerTopology)
+  int firstHBRing_, lastHBRing_;
```

Comment:
Why are these in the DataFormats?
It may be OK to keep them as transients to aid.
Values in per-event persisted objects are meant to be changing event by event and object by object.
Current choice is not appropriate.